src/main/java/com/icthh/xm/ms/entity/web/rest/XmEntityResource.java

```diff
@@ -571,7 +571,14 @@ private void produceEvent(XmEntity entity, String eventType) {
                                                   Map<String, Object> functionInput) {
         Map<String, Object> fContext = functionInput != null ? functionInput : Maps.newHashMap();
         FunctionContext result = functionService.execute(functionKey, IdOrKey.of(idOrKey), fContext);
-        return ResponseEntity.ok().body(result.functionResult());
+
+        ResponseEntity.BodyBuilder response = ResponseEntity.ok();
+
+        if (result.isBinaryData()) {
+            response.header("content-type", result.getBinaryDataType());
```

Comment:
suggest using `org.springframework.http.HttpHeaders#CONTENT_TYPE` instead of constant string

Reply:
surely, used